includes/wikia/DefaultSettings.php

```diff
@@ -630,6 +630,7 @@
  * Includes that are needed for Mercury to work
  */
 
+include_once( "$IP/extensions/wikia/WikiaMobile/WikiaMobile.setup.php" );
```

Comment:
But doesn't Mercury rely on WikiaMobile to remove the inline styles from the markup?